python/ray/util/sgd/torch/training_operator.py

```diff
@@ -930,7 +945,20 @@ def scheduler_step_freq(self):
 
 class CreatorOperator(TrainingOperator):
     """A subclass of TrainingOperator specifically for defining training
-    state using creator functions.
+    state using creator functions. This allows for backwards compatibility
+    with pre Ray 1.0 versions.
```

Comment:
should be 1 line summary. You can break this into two paragraphs. (pydoc style)

Reply:
done